### PR TITLE
CONTRIBUTING.md: add instructions on how to remote desktop into Windows CI

### PR DESCRIPTION
**Pull request by janeyx99**
Adds a link to existing instructions in CONTRIBUTING.md, so those instructions are more visible to contributors.

### CODE REVIEW

**Review by samestep — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1078,3 +1078,7 @@ following steps:
 4. Now you can find the pytorch working directory, which could be
    `~/workspace` or `~/project`, and run commands locally to debug
    the failure.
+
+For certain Windows failures, it may be useful to have a full [Remote
+Desktop](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-clients) connection. See detailed instructions [here](https://github.com/pytorch/pytorch/wiki/Debugging-Windows-with-Remote-Desktop-or-CDB-(CLI-windbg)-on-CircleCI)
+for how to set that up after reaching step 3.
```
Comment (walterddr) [inline CONTRIBUTING.md:1084]: hmm. I am not sure how this instruction links to the 4 steps above. do you mean we should do this after step 1,2,3 (e.g. skip step 4?)

Comment (janeyx99) [reply CONTRIBUTING.md:1084]: Ah, this is all under the heading CI Failure Tips. The above steps show how to rerun, this is an additional step people can do for Windows if they'd like to access more of the machine. I just mean steps 1 and 2 are still necessary for the instructions to work.

Comment (samestep) [reply CONTRIBUTING.md:1084]: Is this perhaps clearer then?
```suggestion
for how to set that up after doing steps 1 and 2.
```

Comment (janeyx99) [reply CONTRIBUTING.md:1084]: Maybe I should just say rerunning the job, because it's not that the user necessarily needs to refer back to those steps.

**Review by walterddr — APPROVED** [review]
lgtm

**Pull request closed without merging**